.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - repo: local
+    hooks:
+      - id: conventional-commits
+        name: Conventional Commit Minder
+        entry: scripts/conventional-commits
```

Comment:
should it be `scripts/` or `contrib/scripts` ?

Reply:
Good catch. I had this in `/scripts/` in my repo; missed that...